__tests__/app.test.js

```diff
@@ -155,3 +155,74 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST /api/articles/:article_id/comments responds with status 201 and the posted comment", () => {
```

Comment:
I would also like to see another test that tests it ignore extra properties (such as name: 'hannah') and still returns a 201